configuration-reference/functions/distinctcountrawhllplus.md

```diff
@@ -0,0 +1,37 @@
+---
+description: >-
+  This section contains reference documentation for the DISTINCTCOUNTRAWHLLPLUS
+  function.
+---
+
+# DISTINCTCOUNTRAWHLLPLUS
+
+Returns HLLPlus response serialized as string. The serialized HLLPlus can be converted back into an HLLPlus and then aggregated with other HLLPluses. A common use case may be to merge HLLPlus responses from different Pinot tables, or to allow aggregation after client-side batching.
+
+## Signature
+
+> DISTINCTCOUNTRAWHLLPLUS(colName)
+> DISTINCTCOUNTRAWHLLPLUS(colName, p)
```

Comment:
Should we name these parameters more explicitly? I believe these refer to 'precision' and 'sparse? precision', would be good to have some basic explanation here.

Reply:
@cbalci i added a one line in the above section to describe p and sp, please take a look, thanks